"create task from email" option should not be there when Tasks app is not installed

### Steps to reproduce

1. have mail app installed but not tasks
2. in a received email, click the ... menu > more actions > create task
3. modal shows, you enter details
4. error: could not create task


### Expected behavior

the option "create task" should only be there when Tasks app is installed


### Actual behavior

4. error: could not create task

### Mail app version

3.7.5

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

NC 29.0.1

Browser console output after clicking "create task" in an email:

```
14:14:44.612 [ERROR] mail: Creating event from message failed 
Object { app: "mail", uid: "admin", level: 2, error: TypeError }
​
app: "mail"
​
error: TypeError: e.calendar.dav is undefined
​
level: 2
​
uid: "admin"
​
<prototype>: Object { … }
​​
__defineGetter__: function __defineGetter__()
​​
__defineSetter__: function __defineSetter__()
​​
__lookupGetter__: function __lookupGetter__()
​​
__lookupSetter__: function __lookupSetter__()
​​
__proto__: 
​​
constructor: function Object()
​​
hasOwnProperty: function hasOwnProperty()
​​
isPrototypeOf: function isPrototypeOf()
​​
propertyIsEnumerable: function propertyIsEnumerable()
​​
toLocaleString: function toLocaleString()
​​
toString: function toString()
​​
valueOf: function valueOf()
​​
<get __proto__()>: function __proto__()
​​
<set __proto__()>: function __proto__()
ConsoleLogger.js:74:18
    value ConsoleLogger.js:74
    value ConsoleLogger.js:100
    onSave TaskModal.vue:201
    VueJS 33

```

--- a/src/components/Envelope.vue
+++ b/src/components/Envelope.vue
@@ -422,6 +422,7 @@
 					{{ t('mail', 'Reply with meeting') }}
 				</ActionButton>
 				<ActionButton
+					v-if="tasksEnabled"
 					:close-after-click="true"
 					@click.prevent="showTaskModal = true">
 					<template #icon>
@@ -899,6 +900,10 @@ export default {
 
 		archiveMailbox() {
 			return this.mainStore.getMailbox(this.account.archiveMailboxId)
+		},
+
+		tasksEnabled() {
+			return this.mainStore.getTaskCalendarsForCurrentUser.length > 0
 		},
 
 		isSnoozedMailbox() {

--- a/src/components/MenuEnvelope.vue
+++ b/src/components/MenuEnvelope.vue
@@ -165,6 +165,7 @@
 				{{ t('mail', 'Reply with meeting') }}
 			</ActionButton>
 			<ActionButton
+				v-if="tasksEnabled"
 				:close-after-click="true"
 				@click.prevent="$emit('open-task-modal')">
 				<template #icon>
@@ -459,6 +460,10 @@ export default {
 
 		hasDeleteAcl() {
 			return mailboxHasRights(this.mailbox, 'te')
+		},
+
+		tasksEnabled() {
+			return this.mainStore.getTaskCalendarsForCurrentUser.length > 0
 		},
 
 		isSnoozedMailbox() {

